change/@ni-nimble-angular-eeee6b1e-cb23-4a3c-ab87-8f1b30f413fe.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
nit: could you copy a bit more of the rationale from the AzDO issue into the PR description? Saves clicks now and if exploring history in the future and makes it more accessible to people without AzDO access.